src/3d/symbols/qgsmesh3dsymbol.h

```diff
@@ -76,6 +76,20 @@ class _3D_EXPORT QgsMesh3DSymbol : public QgsAbstract3DSymbol
     void writeXml( QDomElement &elem, const QgsReadWriteContext &context ) const override;
     void readXml( const QDomElement &elem, const QgsReadWriteContext &context ) override;
 
+    /**
+     * Returns if the 3d rendering is enabled
+     *
+     * \since QGIS 3.14
+     */
+    bool renderer3dEnabled() const;
```

Comment:
would it be possible use enabled() isEnabled() etc consistenty with e.g. https://github.com/qgis/QGIS/blob/b510ad0803383e62a3022ff84914b980df68c9fd/src/core/mesh/qgsmeshrenderersettings.h#L44